### ISSUE TITLE
Documentation doesn't show Dropdown is beta / isn't included in the react-bootstrap

### ISSUE THREAD

**Issue by ianchanning** [react-bootstrap/react-overlays#280]
**tl;dr: Would it be possible to put a label on the dropdown component in the documentation or something to indicate that it's not currently included in react-bootstrap library: https://react-bootstrap.github.io/react-overlays/#dropdown**

* react-overlays version: v1.1.1
* react-bootstrap version: v0.32.4

Firstly I want to say that the upcoming Dropdown component is brilliant. It's great that you offer a simplified underlying component. 

However I hadn't realised that it was in beta and so isn't currently included the in current 0.32.4 react-bootstrap library and I've just spent two days through the very cryptic messages that React gives out trying to implement the Dropdown component:

```
Warning: React.createElement: type is invalid -- expected a string (for built-in components) 
or a class/function (for composite components) but got: undefined. You likely forgot to 
export your component from the file it's defined in, or you might have mixed up default 
and named imports.
```

### Background

We are trying to migrate away from using the full blown react-bootstrap library as we have a lot of our own styles. The select HTML element is one main component that you can't just style with CSS along because the options elements are controlled by the OS / browser itself.

If not using react-bootstrap for dropdowns the two main options that I found were:

1. https://github.com/JedWatson/react-select - but this component on it's own manages to make itself almost as complex as the entire react-bootstrap library
2. https://github.com/fraserxu/react-dropdown - but the code for this raises multiple flags with the way that it uses React

The react-overlays Dropdown component is pretty perfect because it's a very simple component that get's heavily tested as it's underlying the react-bootstrap library but allows us to entirely implement our own styles on top.

### Problems

But two things I didn't realise

1. It's a beta component which means that it's not in the current react-bootstrap (v0.32.4)
2. You can't include react-overlays as well as react-bootstrap because the two libraries conflict (the fact that it took me 2 days to realise this makes me question my sanity)

### Simplest way out of the confusion I can think of

Would it be possible to add a tag in the documentation to show that the component there was in Beta - you could also apply this to any new components you will add in the future. I might then have been a bit more cautious about trying to implement something based on it.

**Update**: I guess it's not beta as this is currently at v1.1.1 but it's only included in the beta release of react-bootstrap any indication of this would be helpful.

**Comment by taion** [comment]
You can use both versions of react-overlays just fine. It's a direct dependency, not a peer dependency.
